pkg/client/restclient/request.go

```diff
@@ -624,7 +624,7 @@ func (r *Request) tryThrottle() {
 		r.throttle.Accept()
 	}
 	if latency := time.Since(now); latency > longThrottleLatency {
-		glog.Warningf("Throttling request took %v, request: %s:%s", latency, r.verb, r.URL().String())
+		glog.V(2).Infof("Throttling request took %v, request: %s:%s", latency, r.verb, r.URL().String())
```

Comment:
at the beginning I did use V(4), after I take a deep look at glog, and also as the original Warningf usage, finally I use V(2), ` we really just want it in e2e clusters` we should use V(4) here.